FWCore/Framework/src/StreamSchedule.cc

```diff
@@ -143,6 +143,11 @@ namespace edm {
 
       return workerManager.getWorker(*modpset, preg, prealloc, processConfiguration, moduleLabel);
     }
+
+    template <typename T>
+    auto findConditionalTaskModulesBegin(T& modnames) {
+      return std::find(modnames.begin(), modnames.end(), "#");
```

Comment:
@Dr15Jones Both callers use the range `findConditionalTaskModulesBegin(modnames)+1` to `std::prev(modnames.end())`. What do you think, should this function
* return the iterator to position after the `#`, or
* return the range from position after the `#` to the `end()-1`

instead of the minimal abstraction done here?

Reply:
I think the range